src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form
+      onsubmit="onSubmit()"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <div class="pesonalInfo">
+        <fieldset class="fieldset-block">
+          <legend>Personal information</legend>
+          <div class="form-field">
+            Surname:
+            <input
+              class="form-input"
+              type="text" name="surname"
+              id="surname"
+              autocomplete="off">
+          </div>
+          <div class="form-field">
+            Name:
+            <input
+              class="form-input"
+              type="text" name="name"
+              id="name"
+              autocomplete="off"
+              >
```

Comment:
```suggestion
            <input
              class="form-input"
              type="text" name="name"
              id="name"
              autocomplete="off"
            >
```
Закриваюча дужка має бути на тому ж рівні, що і відкриваюча